spec/Vehicle/Vehicle.vspec

```diff
@@ -399,10 +399,48 @@ CurrentLocation.Altitude:
   datatype: double
   type: sensor
   unit: m
-  description: Current altitude relative to WGS 84 reference ellipsoid.
+  description: Current altitude relative to WGS 84 reference ellipsoid, as measured at position of GNSS receiver antenna.
 
 CurrentLocation.VerticalAccuracy:
   datatype: double
   type: sensor
   unit: m
   description: Accuracy of altitude.
+
+CurrentLocation.GNSSReceiver:
+  type: branch
+  description: Information on the GNSS receiver used for determining current location.
+
+CurrentLocation.GNSSReceiver.Status:
+  datatype: string
+  type: sensor
+  allowed: ['NO_FIX', 'FIX_2D', 'FIX_3D']
+  description: Fix status of GNSS receiver.
+
+CurrentLocation.GNSSReceiver.MountingPosition:
+  type: branch
+  description: Mounting position of GNSS receiver antenna relative to vehicle coordinate system.
+               Axis definitions according to ISO 8855. Origo at center of (first) rear axle.
+
+CurrentLocation.GNSSReceiver.MountingPosition.X:
+  datatype: int16
+  type: attribute
+  description: Mounting position of GNSS receiver antenna relative to vehicle coordinate system.
+               Axis definitions according to ISO 8855. Origo at center of (first) rear axle.
+               Positive values = forward of rear axle. Negative values = backward of rear axle.
+
+CurrentLocation.GNSSReceiver.MountingPosition.Y:
+  datatype: int16
+  type: attribute
+  description: Mounting position of GNSS receiver antenna relative to vehicle coordinate system.
+               Axis definitions according to ISO 8855. Origo at center of (first) rear axle.
+               Positive values = left of vehicle center. Negative values = right of vehicle center.
```

Comment:
I'm not sure if it helps to introduce the term vehicle center here...

Reply:
You mean describing what vehicle center is, e.g. "center point on rear axle" or similar

Reply:
Maybe I misunderstood your comment. Based on what was discussed at the meeting I assume that the coordinates (x=0,y=0,z=0) means center of first rear axle. And then I try to describe that a positive y means a position left of origin, as seen by a person looking forward. 
But the term "vehicle center" might be confusing, is a text like below better:


```
Axis definitions according to ISO 8855. Origin at center of (first) rear axle.
Positive values = left of origin. Negative values = right of origin
Left/Right is as seen from driver perspective, i.e. by a person looking forward.
```

Reply:
Looks better now :)